rubicon_ml/client/rubicon.py

```diff
@@ -143,18 +143,21 @@ def get_project(self, name=None, id=None):
 
         if name is not None:
             return_err = None
+
             for repo in self.repositories:
                 try:
                     project = repo.get_project(name)
                 except Exception as err:
                     return_err = err
                 else:
-                    project = Project(project, self.config)
-                    return project
-            raise RubiconException("all configured storage backends failed") from return_err
+                    return Project(project, self.config)
+
+            if len(self.repositories) > 1:
+                raise RubiconException("all configured storage backends failed") from return_err
+            else:
+                raise return_err
```

Comment:
Should this also use the new method?

Reply:
`Rubicon` doesn't extend from `Base` (even though it reimplements nearly everything on it) - I'm gonna open an issue to look into that

Reply:
#387